pkg/kubelet/dockershim/helpers.go

```diff
@@ -30,6 +30,10 @@ import (
 	runtimeApi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+const (
+	annotationPrefix = "annotation."
```

Comment:
We can do that, but I was afraid that this would grow too long. E.g., the annotation key could be `io.kubernetes.terminationmessage`. After this, it becomes `io.kubernetes.annotation.io.kubernetes.terminationmessage`